mdb-coverage-table.html

```diff
@@ -336,6 +337,10 @@
         'select-column': '_selectedChanged'
       },
 
+      _columnsShownRecAll: function () {
+          this.fire('columnsShownRecAll', {"columnsShownRecAll": this.columnsShown});
+      },
+
```

Comment:
we might consider adding { notify: true } to the "columnsShown" property and use polymer default event "columns-shown-changed". Merging this for now.